webpack/assets/javascripts/react_app/stores/NotificationsStore.js

```diff
@@ -1,10 +1,11 @@
 import AppDispatcher from '../dispatcher';
-import { ACTIONS } from '../constants';
+import { ACTIONS, STATUS } from '../constants';
 import AppEventEmitter from './AppEventEmitter';
 import moment from 'moment';
 
 let _notifications = {};
 let _expandedTab = null;
```

Comment:
No. _expandedTab here refers to which of the accordion tabs (group) in the drawer is expanded